### ISSUE TITLE
Menu height

### ISSUE THREAD

**Issue by petebocken** [insidegui/PodcastMenu#35]
Would be great if the default height of the menu was higher or could be resized manually higher.  That way you would not have to scroll down to click the controls.

**Comment by insidegui** [comment]
I agree. I'm working on an update that changes the playback UI quite a bit, hopefully I can address this for this update as well.

**Comment by petebocken** [comment]
Also remember that width/height after a quit/open.
